tools/install-monorepo.sh

```diff
@@ -1,9 +1,11 @@
 #!/usr/bin/env bash
 
-set -eo pipefail
+# Start in the monorepo root folder.
+root_dir="$(dirname "${BASH_SOURCE[0]}")/.."
```

Comment:
Technically they do give slightly different results. I actually find it much easier to troubleshoot when the `..` remains. Obviously the current instance is fairly simple and either way works well, but imagine the following contrived example:

Directory structure:
```
/home/jpuser/
    jetpack/projects/draco/
        test.sh
    draco_symlink -> jetpack/projects/draco/
```

`test.sh`:
```
root_dir="$(dirname "${BASH_SOURCE[0]}")/.."
root_dir2="$(cd "${BASH_SOURCE[0]}"/..&& pwd)"
echo $root_dir
echo $root_dir2
```

Standing in `/home/jpuser/`, I get the following:
```
./draco_symlink/..
/home/jpuser
```

Standing in `/home/jpuser/draco_symlink`, I get the following:
```
./..
/home/jpuser
```

Standing in `/home/jpuser/jetpack/projects/draco`, I get the following:
```
./..
/home/jpuser/jetpack/projects
```

For me, troubleshooting with the relative output would be much easier to follow vs. the latter; I immediately see there's a problem with it accessing a relative parent dir.

Happy to change it if you disagree, but otherwise I'll leave this.

Reply:
> ```
> ./draco_symlink/..
> /home/jpuser
> ```

The main reason many of the scripts set and use a `BASE` variable is so that things can work even if something somewhere does a `cd`, e.g. to conveniently run various `composer` commands from within a project. The relative version would be useless for that situation.

> Happy to change it if you disagree, but otherwise I'll leave this.

Since in this case we don't need the variable at all, we could avoid the debate by getting rid of it entirely and just doing like this: 🙂
```
cd "$(dirname "${BASH_SOURCE[0]}")/.."
```

Reply:
🤝 6a6a1b4767